### DISCUSSION TITLE
CrossEntropy loss

### DISCUSSION THREAD

**Discussion by Sarmadfismael** [qubvel/segmentation_models.pytorch#469] [Q&A] [answered]
hi all 
How can I use the cross-entropy loss in SMP.losses?

**Comment by Sarmadfismael** [comment, marked as answer]
I found the solution 
By using the Focal loss and put the gamma = 0
